deploy/vagrant/conf/zookeeper.yml

```diff
@@ -0,0 +1,7 @@
+# Release means using binary distribution from http://archive.apache.org/dist/zookeeper
+# {Release}
+Type: Release
```

Comment:
Added "Note that zookeeper will only be downloaded and set up when "Masters" in tachyon.yml is larger than 1."